src/gov-action-contracts/address-registries/L2AddressRegistry.sol

```diff
@@ -7,11 +7,13 @@ contract L2AddressRegistry is IL2AddressRegistry {
     IL2ArbitrumGoverner public immutable coreGov;
     IL2ArbitrumGoverner public immutable treasuryGov;
     IFixedDelegateErc20Wallet public immutable treasuryWallet;
+    IArbitrumDAOConstitution public immutable arbitrumDAOConstitution;
```

Comment:
thoughts on including the Foundation vesting wallet here? prob not needed

Reply:
yeah i don't think we need it